postfixCalculator/postfixCalculator.c

```diff
@@ -0,0 +1,83 @@
+﻿#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include "postfixCalculator.h"  
+#include "stack.h"
+
+bool isDigit(char element) {
+    return element == '0' || element == '1' || element == '2' || element == '3' || element == '4' ||
+        element == '5' || element == '6' || element == '7' || element == '8' || element == '9';
```

Comment:
```suggestion
    return element >= '0' && element <= '9';
```
Ещё есть стандартная библиотечная функция isdigit